test/api/bootstrap.test.js

```diff
@@ -1,19 +1,29 @@
-Promise.props = require("promise-props")
+Promise.props = require('promise-props');
 
-const AWSMocks = require("./support/aws-mocks")
+const AWSMocks = require('./support/aws-mocks'); // eslint-disable-line no-unused-vars
```

Comment:
since the actual variable is never used, maybe we could just leave out the assignment and the eslint-disable:

```js
require('./support/aws-mocks');
```

Reply:
Thats a better idea. We might actually be able to do that in the `models/index` file as well, since we require all the models just to inject them into the global scope.